### PR TITLE
Allow input from stdin

### PR DESCRIPTION
**Pull request by JonnyBurger**
This allows for cool things such as easy YouTube playback using [ytdl](https://github.com/fent/node-ytdl)

youtube open-pip

I even made another module called [twitch-url-cli](https://github.com/jonnyburger/twitch-url-cli) which makes easy Twitch playback possible.

twitch

If you are OK with it, I can even make a PR for the README about ytdl and my module.

### CODE REVIEW

**Comment by albinekb** [comment]
This is awesome! 🎉 

Maybe we should support just pasting an youtube URL?
Include both twitch-url and ytdl as dependencies to this one?

File: index.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env node
 const open = require('open-pip')
 const ora = require('ora')
+const getStdin = require('get-stdin')
 const spinner = ora().start()
 
-const input = process.argv[2]
+getStdin()
+.then(stdin => {
+	if (stdin) return stdin.trim();
```
Comment (albinekb) [inline index.js]: remove semicolon 👌 


I'll set up the linter to run in travis when i get home 🎉

**Comment by JonnyBurger** [comment]
Semi removed.
Yeah, we could do that. We could use the following module: [get-youtube](https://www.npmjs.com/package/get-youtube-id)
For Twitch, I think we would need to write our own parser.

**Comment by albinekb** [comment]
Awesome, let's merge this for now as `stdin` is 👌

**Comment by albinekb** [comment]
Released as `1.2.0` @JonnyBurger thanks again 🎉

**Comment by albinekb** [comment]
Would be nice to update the readme with examples for this 👌

**Pull request merged**